src/Costura.Fody/NetStandardAssemblyResolver.cs

```diff
@@ -0,0 +1,74 @@
+﻿using System;
+using System.Collections.Generic;
+using Mono.Cecil;
+
+/// <summary>
+/// An assembly resolver that resolves with the <see cref="IAssemblyResolver"/> of the <see cref="ModuleDefinition"/>
+/// of the given <see cref="ModuleWeaver"/> with special handling for the <c>netstandard</c> assembly reference to
+/// support .NET Framework 4.7 and lower.
+/// </summary>
+public class NetStandardAssemblyResolver : IAssemblyResolver
```

Comment:
I think the naming is a bit confusing. It threw me off a bit because I thought it was primarily targeted at netstandard. Not sure what a good name would be though.